src/components/NewWorkspaceModal.js

```diff
@@ -380,12 +383,30 @@ const NewWorkspaceModal = withDisplayName(
                     }),
                   ]),
               ]),
+              isGoogleBillingProject() &&
+                div({ style: { margin: '1rem 0.25rem 0.25rem 0' } }, [
+                  h(
+                    LabeledCheckbox,
+                    {
+                      style: { margin: '0rem 0.25rem 0.25rem 0' },
+                      checked: enhancedBucketLogging,
+                      disabled: !!requireEnhancedBucketLogging || groups.length > 0,
+                      onChange: () => setEnhancedBucketLogging(!enhancedBucketLogging),
+                    },
+                    [label({ style: { ...Style.elements.sectionHeader } }, ['Workspace will have protected data'])]
+                  ),
+                  h(InfoBox, { style: { marginLeft: '0.25rem', verticalAlign: 'middle' } }, [
+                    'If checked, Terra will log all data access requests to the workspace bucket. ' +
+                      'This feature is automatically enabled when a workspace is created with Authorization Domains.',
+                  ]),
+                  p({ style: { marginTop: '.25rem' } }, ['Access to data will be logged by Terra']),
```

Comment:
Since "Access to data will be logged by Terra" is additional information that is useful to understand when selecting "Workspace will have protected data", I would suggest that you link it to the checkbox itself for the benefit of screen readers. This can be accomplished in the following manner (it's fine for a control to have both a label and additional description(s), and the user will be notified that there is additional content they can read), where `aria-describedby` is used to do the linking.

```
isGoogleBillingProject() &&
                div({ style: { margin: '1rem 0.25rem 0.25rem 0' } }, [
                  h(IdContainer, [
                    (id) =>
                      h(Fragment, [
                        h(
                          LabeledCheckbox,
                          {
                            style: { margin: '0rem 0.25rem 0.25rem 0' },
                            checked: enhancedBucketLogging,
                            disabled: !!requireEnhancedBucketLogging || groups.length > 0,
                            onChange: () => setEnhancedBucketLogging(!enhancedBucketLogging),
                            'aria-describedby': id,
                          },
                          [label({ style: { ...Style.elements.sectionHeader } }, ['Workspace will have protected data'])]
                        ),
                        h(InfoBox, { style: { marginLeft: '0.25rem', verticalAlign: 'middle' } }, [
                          'If checked, Terra will log all data access requests to the workspace bucket. ' +
                            'This feature is automatically enabled when a workspace is created with Authorization Domains.',
                        ]),
                        p({ id, style: { marginTop: '.25rem' } }, ['Access to data will be logged by Terra']),
                      ]),
                  ]),
                ]),
 ```               
              
Theoretically the same is true for "Authorization domain" and "Additional group management controls", but the custom select component makes it more difficult because `aria-describedby` won't be passed though to the underlying controls… and to be honest, I don't think "Additional group management controls" provides any useful information. So I wouldn't bother with that one.

Reply:
Good eye for accessibility! Updated from your example. 👍